.changeset/tall-rockets-hide.md

```diff
@@ -0,0 +1,2 @@
+---
+---
```

Comment:
Empty?

Reply:
Oops good catch - fixed. https://github.com/FuelLabs/fuels-ts/pull/2691/commits/ba09ad9a805a8db315b68f4f9acbd9b0e9ebb1fc

Reply:
Does this not need a `create-fuels` bump?

Reply:
https://github.com/FuelLabs/fuels-ts/pull/2691/commits/1dda01396dc321f0f4684ad5d0d2156c9ddb636d

Fixed